src/OpenTelemetry.AutoInstrumentation.Native/cor_profiler.cpp

```diff
@@ -1770,617 +1796,1418 @@ HRESULT CorProfiler::RunAutoInstrumentationLoader(const ComPtr<IMetaDataEmit2>&
     return S_OK;
 }
 
-HRESULT CorProfiler::GenerateLoaderMethod(const ModuleID module_id, mdMethodDef* ret_method_token)
+HRESULT CorProfiler::ModifyAppDomainCreate(const ModuleID module_id, mdMethodDef patch_app_domain_setup_method)
```

Comment:
It would be nice to have some comments that explain what this method is trying to do, especially since it's a longer function and it is modifying IL.